jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -1229,15 +1237,17 @@ def _make_conv_harness(name, *, lhs_shape=(2, 3, 9, 10), rhs_shape=(3, 3, 4, 5),
     # TODO(bchetioui): the NCDHW data format is not supported on CPU for TF
     # for now. That path is thus disabled to allow the code to use XLA instead.
   ]
+  for enable_xla in [False, True]
 ) + tuple(
     # tf.nn.convolution only supports a subset of the possible dtypes for JAX
     # convolutions (float16, float32, float64). With the below tests, we ensure
     # that we avoid this branch in cases when it would not succeed.
     _make_conv_harness("tf_conversion_path_dtype", lhs_shape=lhs_shape,
                      padding='VALID', rhs_shape=rhs_shape, dtype=dtype,
                      dimension_numbers=dimension_numbers, lhs_dilation=(1, 1),
-                     rhs_dilation=(1, 1))
-  for dtype in jtu.dtypes.all_inexact
+                     rhs_dilation=(1, 1), enable_xla=False,
+                     expect_xla_disabled_exception=True)
```

Comment:
Why is there an exception here?

Reply:
These cases where we expect an exception were intended as checks that we properly fell back to XLA when the TF conversion could not be achieved. Since there is no scenario when we fallback anymore, it is not that useful. I will remove them.